nautobot/core/views/generic.py

```diff
@@ -205,13 +206,11 @@ def get(self, request):
         filter_form = None
         hide_hierarchy_ui = False
         clear_view = request.GET.get("clear_view", False)
+        view_name = self.view_name or validated_viewname(model, "list")
```

Comment:
Wonder if we could use [`resolve(request.path).url_name`](https://docs.djangoproject.com/en/4.2/ref/urlresolvers/#resolve) as a "smarter" solution in general here that might remove the name for the `view_name` parameter altogether: